compiler/assemble.js

```diff
@@ -112,6 +144,8 @@ export default (funcs, globals, tags, pages, data, flags, noTreeshake = false) =
     encodeVector(funcs.map(x => getType(x.params, x.returns))) // type indexes
   );
 
+  const nameSection = customSection('name', encodeNames(funcs));
```

Comment:
I like to keep the Wasm binaries small/simple by default, could we have this behind some new general "debug output" pref, maybe something short just like `-d` (`Prefs.d`)?